test/basic/smack_code_annot.c

```diff
@@ -0,0 +1,43 @@
+#include "smack.h"
+
+// @expect verified
+// @flag --bit-precise --float
```

Comment:
I don't think we should be adding regressions that require `--float` flag into `basic` folder. I guess we should add those into `float` folder. The same goes for `--bit-precise`. Maybe a good idea would be to split the benchmark into the int and float parts, and put the into one into bit vectors folder, and the float one into the float folder. Or keep it as one regression and put it into float folder.

Reply:
The reason that these flags are added is because we don't distinguish different integer types or float/double types without these flags. Namely, in order to test this feature, we need these flags to perform meaningful type checking.  So I think that it's fine we add these flags to benchmarks in the `basic` folder as long as these flags are not used to test the features they enable.

Reply:
We should not have any benchmarks in the `basic` folder that require the theory of bitvectors or floating-points. When you add `--float` and `--bit-precise` that implies you need both of these theories. No?

Reply:
I split the tests into two copies for each folder.